### ISSUE TITLE
add plugin to block voice spamming

### ISSUE THREAD

**Issue by wopox1337** [wopox1337/ChatsAdditions_AMXX#36]

**Comment by Nord1cWarr1or** [comment]
Что подразумевается под спаммингом? Слишком долгая работа микрофона одного юзера?

**Comment by wopox1337** [comment]
@Nord1cWarr1or да, флуд скорее, когда игрок мешает игрока другим своими разговорами.

**Comment by Nord1cWarr1or** [comment]
@wopox1337, тогда следует обратить внимание на то, что например на моём сервере игроки иногда включают музыку через стерео микшер + войс.

Конечно, я могу просто не ставить данный будущий плагин, но можно сделать например, если игрок превышает лимит времени выделенного на микрофон, у всех начинается автоматическое голосование за гаг данного игрока.

**Comment by wopox1337** [comment]
@Nord1cWarr1or интересная идея. Именно для этого и созданы комментарии, чтобы обсуждать 👍🏼
